xwiki-platform-core/xwiki-platform-model/xwiki-platform-model-validation/xwiki-platform-model-validation-api/src/main/java/org/xwiki/model/validation/EntityNameValidationManager.java

```diff
@@ -51,7 +52,9 @@
     Set<String> getAvailableEntityNameValidations();
 
     /**
-     * Allow to reset the configuration of the strategies.
+     * @return the list of available {@link EntityNameValidation} components.
+     * @since 13.4
+     * @since 12.10.8
      */
-    void resetStrategies();
+    List<EntityNameValidation> getAvailableEntityNameValidationsComponents();
```

Comment:
Not sure to understand the purpose of this method, you could achieve the same by calling the component manager and getting the list of components of type `EntityNameValidation` no?

Reply:
AFAICS that's even exactly your implem so I don't see why it needs to be declared in the API.